docs/accessibility-statement.md

```diff
@@ -4,45 +4,45 @@ title: Accessibility Statement
 
 _Last updated: April 8, 2019_
 
-We want everyone who uses Gatsby to feel welcome and find the experience rewarding: this includes developers with disabilities and users with disabilities for Gatsby sites, docs, and resources.
+हम चाहते हैं कि सभी लोग जो Gatsby का स्वागत करने और अनुभव को पुरस्कृत करने का अनुभव करते हैं: इसमें Gatsby साइटों, डॉक्स और संसाधनों के लिए अक्षमताओं वाले उपयोगकर्ता और इस के साथ डेवलपर्स शामिल है।
 
-This page was created to collect accessibility information about the Gatsby ecosystem in one place and provide communication channels for people with disabilities to get help with Gatsby.
+यह पेज एक स्थान पर गैट्सबी पारिस्थितिक तंत्र के बारे में जानकारी प्राप्त करने और विकलांग लोगों के लिए संचार चैनल प्रदान करने के लिए गैट्सबी की सहायता प्राप्त करने के लिए बनाया गया था।
```

Comment:
please recheck this sentence translation, this does not sounds correct

Reply:
you missed this. Please revisit at this page translation.

Reply:
```
ये पेज Gatbsy इकोसिस्टम की एक्सेसिबिलिटी जानकारी को एक जगह इकट्ठा करने और अक्षमताओं वाले लोगों को Gatsby से मदद लेने के लिए संचार चैनल देने के लिए बनाया गया था। 
```

let's use this.

Reply:
One more minor mistake here. You forgot to add a line break at line 10 after copy-pasting this.
Please take a moment to check if nothing else is broken due to the changes :)